src/test/java/org/example/AppTest.java

```diff
@@ -1,44 +1,32 @@
 package org.example;
 
-import org.junit.Test;
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Assert;
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.annotations.Test;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.openqa.selenium.remote.DesiredCapabilities;
-
-import java.util.concurrent.TimeUnit;
-
+import org.testng.annotations.BeforeTest;
 
 public class AppTest {
-    @Test
-    public void RozetkaTest() {
-        DesiredCapabilities capabilities = DesiredCapabilities.chrome();
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("incognito");
-        capabilities.setCapability(ChromeOptions.CAPABILITY, options);
+    WebDriver browser;
+
+    @BeforeTest
+    public void preCondition() {
         System.setProperty("webdriver.chrome.driver", "d:\\Downloads\\chromedriver_win32\\chromedriver.exe");
-        WebDriver browser = new ChromeDriver(capabilities);
+        browser = new ChromeDriver();
         browser.get("https://www.rozetka.com.ua/");
+        JavascriptExecutor js = (JavascriptExecutor) browser;
```

Comment:
please all 'low-level' code into corresponding page object, you should avoid operations with webdriver in your test. Test should contain only methods from pageobjects and assertions

Reply:
I have cretated new PageObj for WebPage start and WebPage Scroll